packages/ticker/src/TickerListener.js

```diff
@@ -27,7 +27,7 @@ export default class TickerListener
         /**
          * The calling to execute.
          * @private
-         * @member {Function}
+         * @member {object}
```

Comment:
These should be `any` or `*`. When you use `object` it means `{}` specifically.

Reply:
OK